object/storage.go

```diff
@@ -17,24 +17,15 @@ package object
 import (
 	"bytes"
 	"fmt"
-
 	awss3 "github.com/aws/aws-sdk-go/service/s3"
 	"github.com/casbin/casdoor/util"
+	"github.com/casbin/casdoor/util/filesystem"
 	"github.com/qor/oss"
 	"github.com/qor/oss/aliyun"
-	//"github.com/qor/oss/qiniu"
```

Comment:
Please set your Goland -> Editor -> Code Style -> Go like this: 
![image](https://user-images.githubusercontent.com/42638489/128595259-d7089d4e-c62b-4f43-9c80-c2d75373a2d4.png)